packages/client/src/generation/generateClient.ts

```diff
@@ -112,7 +111,7 @@ export async function buildClient({
   const defaultClient = new TSClient({
     ...nodeClientOptions,
     reusedTs: 'index',
-    reusedJs: 'index',
+    reusedJs: '.',
```

Comment:
Forces it through the exports map, this is temporary. When we GA, this `default` client will be deleted/replaced.